### PR TITLE
Implement get tweet age

### PR DESCRIPTION
**Pull request by JVieira13**
Just implemented a code (and respective tests) that gives us a tweet age.

### CODE REVIEW

**Review by m-maga — APPROVED** [review]

**Review by h314to — APPROVED** [review]
Good job! Good to merge.

File: Main.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from datetime import datetime as dt
+
+
```
Comment (alesr) [inline Main.py]: Please remove this extra blank line.

File: Tests.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import unittest
+import Main as mm
+
+
```
Comment (alesr) [inline Tests.py]: Same here. Please remove this extra blank line.

**Review by alesr — COMMENTED** [review]
The file names should start with lowercase letter.
i.e.: `main.py` and `tests.py`.

File: Main.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    date_format = "%m-%d-%Y"
+    if todayDate == "":
+        td = dt.now()
+        todayDate = td.strftime(date_format)
+    a = dt.strptime(lastTweetDate, date_format)
+    b = dt.strptime(todayDate, date_format)
+    delta = b - a
+    return delta.days
+
+print lastTweet("2-6-2017")
```
Comment (coppetti) [inline Main.py]: last print could be removed. 
Logic should be coded outside main
file names should start with lowercase in python: main.py
function names are lowercase with _: last_tweet

File: Tests.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+class testMainMethods(unittest.TestCase):
```
Comment (coppetti) [inline Tests.py]: test_main_methods

**Review by coppetti — CHANGES_REQUESTED** [review]

**Pull request merged**